packages/components/src/components/button-toggle/button-toggle.scss

```
@@ -58,6 +58,12 @@
             &__button--active {
                 background-color: rgba(0, 0, 0, 0) !important;
                 z-index: 1;
+                padding: 0.8rem 0.6rem;
+                white-space: normal;
+                /* postcss-bem-linter: ignore */
+                span {
```

```suggestion
               .dc-btn__text {
```
and maybe you don't need to disable the rule here: `/* postcss-bem-linter: ignore */`